src/App.js

```diff
@@ -32,10 +32,12 @@ function App() {
           <Route exact path="/universities" element={<UniversitySearch />} />
           <Route exact path="/get_uni/:id/:page" element={<ViewUni />} />
           <Route exact path="/course/home" element={<CourseHome />} />
+          <Route exact path="/uwcourse/:id" element={<CoursePage />} />
           <Route
             exact
-            path="/get_uni/:id/:page/:course_name"
-            element={<CoursePage />}
+            path="/course/prevSequence"
+            element={<PreviousCourseSequences />}
```

Comment:
I think this shouldn't be added (PreviousCourseSequences)

Reply:
resolved